src/assignment/views/AssignmentCreate.tsx

```diff
@@ -1,6 +1,10 @@
+import './AssignmentCreate.scss';
+import './AssignmentPage.scss';
+
 import { yupResolver } from '@hookform/resolvers/yup';
 import { Button, Container, Icon, IconName, Spacer } from '@viaa/avo2-components';
 import { type Avo } from '@viaa/avo2-types';
+import { type LomFieldSchema } from '@viaa/avo2-types/types/lom';
```

Comment:
don't do deep imports from libraries
use:
```suggestion
import { type Avo } from '@viaa/avo2-types';
```

and then
```
Avo.Lom.LomField
```

so there is a clear "public interface" of the library, and the library can change folder structure without breaking depedent sites